challenge_cleanup/src/challenge_cleanup/self_cleanup.py

```diff
@@ -331,21 +337,31 @@ def __init__(self, robot, selected_entity_designator, room_des):
-                                                            trash_place_pose,
-                                                            ds.OccupiedArmDesignator(robot, {},
-                                                                                  name="occupied_arm_designator")),
+                                   robot_smach_states.Place(
+                                       robot,
+                                       selected_entity_designator,
+                                       trash_place_pose,
+                                       ds.OccupiedArmDesignator(
+                                           robot,
+                                           arm_properties={"required_trajectories": ["prepare_place"],
+                                                           "required_gripper_types": [arms.GripperTypes.GRASPING]},
+                                           name="occupied_arm_designator")),
                                    transitions={"done": "SAY_PLACE_SUCCESS",
                                                 "failed": "SAY_PLACE_FAILED"})
 
-            smach.StateMachine.add('PLACE_TO_STORE', robot_smach_states.Place(robot, selected_entity_designator,
-                                                                              store_entity_des,
-                                                                              ds.OccupiedArmDesignator(robot, {}, name=
-                                                                                "occupied_arm_designator"),
-                                                                                "on_top_of"),
-                                   transitions={"done": "SAY_PLACE_SUCCESS",
-                                                "failed": "SAY_PLACE_FAILED"})
+            smach.StateMachine.add('PLACE_TO_STORE',
+                                   robot_smach_states.Place(robot,
+                                                            selected_entity_designator,
+                                                            store_entity_des,
+                                                            ds.OccupiedArmDesignator(
+                                                                robot,
+                                                                arm_properties={
+                                                                    "required_trajectories": ["prepare_place"],
+                                                                    "required_gripper_types": [
+                                                                        arms.GripperTypes.GRASPING]},
+                                                                name="occupied_arm_designator"),
+                                                            "on_top_of"),
+                                   transitions={"done": "SAY_PLACE_SUCCESS", "failed": "SAY_PLACE_FAILED"})
```

Comment:
The line is too long, and you break it too late, so you get long line at the top, and then everything pushed at the right side in very short sentences, with a large amount of unused space at the left side.

Breaking this up in smaller pieces makes it much nicer:
```
arm_properties={"required_trajectories": ["prepare_place"],
                           "required_gripper_types": [arms.GripperTypes.GRASPING]}
designator = ds.OccupiedArmDesignator(robot, arm_properties=arm_properties,
                                       name="occupied_arm_designator")
placement = robot_smach_states.Place(robot, selected_entity_designator, store_entity_des,
                                     designator, "on_top_of")
smach.StateMachine.add('PLACE_TO_STORE', placement,
                       transitions={"done": "SAY_PLACE_SUCCESS", "failed": "SAY_PLACE_FAILED"})
```